packaging/installer/README.md

```diff
@@ -293,6 +293,28 @@ To apply the changes you made, you have to restart Netdata.
 
 ---
 
+### Binary Packages
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+Netdata is proud to present it's own flavour of binary packages for the most common operating systems that comply with .RPM and .DEB packaging formats.
```

Comment:
No need for the `Netdata is proud to present `

Reply:
done